adobe/copy_on_write.hpp

```diff
@@ -10,7 +10,225 @@
 
 /**************************************************************************************************/
 
-#include <stlab/copy_on_write.hpp>
+// Trying to fix a few issues in the adobe_source_libraries develop branch without having to
```

Comment:
You only have to pull a single file from stlab - please just add the single file dependency. Overtime more of the asl sources will get moved to stlab.